nautobot/ipam/templates/ipam/namespace_ipaddresses.html

```diff
@@ -0,0 +1,11 @@
+{% extends 'generic/object_retrieve.html' %}
```

Comment:
If these inherit instead from `ipam/namespace.html` (or `ipam/namespace_retrieve.html`) instead of the generic template, does that resolve the tab issue? IIRC that's how the extra tabs on the Device view are currently implemented.

Reply:
Still curious on this point

Reply:
It seems like after restoring and inheriting from `ipam/namespace_retrieve.html`, the tab issue still persists. I think the issue is that we are missing the `"object_detail_content"` key in the context for those custom action view.

Reply:
Yeah, I do think we're going to need a way to specify an object_detail_content per detail action, rather than just one for the whole viewset.